roles/edpm_multipathd/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+edpm_container_cli: "{{ container_cli | default('podman') }}"
```

Comment:
Should we support other container engines than podman given that the targets should be EL9 nodes?

Reply:
I'm not sure how to respond. The general "container_cli" stuff originated in tripleo-ansible, where it was necessary at one point to support both docker and podman. The edpm_container_manage and edpm_container_rm roles do something similar.

Reply:
let's leave it as is, we can talk about it and create a followup PR later.